src/Microsoft.PowerApps.TestEngine.Tests/ConsoleOutputTests.cs

```diff
@@ -139,5 +139,41 @@ public void TestTestCaseEndFailed()
             // Assert that the expected output matches the console output of the function
             Assert.Contains(expected, printer.ToString());
         }
+
+        [Fact]
+        public void TestMultipleBrowserRuns()
+        {
+            var printer = new StringWriter();
+
+            // Set output
+            Console.SetOut(printer);
+
+            // Run tests on first Browser
+            _testEngineEventHandler.SetAndInitializeCounters(2);
+            _testEngineEventHandler.SuiteBegin("TestSuiteName", "./testDirectory", "Chromium", "make.powerapps.com/testapp&source=testengine");
+            _testEngineEventHandler.TestCaseBegin("Case1");
+            _testEngineEventHandler.TestCaseEnd(true);
+            _testEngineEventHandler.TestCaseBegin("Case2");
+            _testEngineEventHandler.TestCaseEnd(false);
+            _testEngineEventHandler.SuiteEnd();
+
+            // Run tests on second Browser
+            _testEngineEventHandler.SetAndInitializeCounters(2);
```

Comment:
Is this going to cause an issue when tests are run asynchronously?

Reply:
Nope, this is basically invoked for each browser test run.  
<img width="784" alt="image" src="https://github.com/microsoft/PowerApps-TestEngine/assets/98551644/dd5b5b67-2fab-4c50-9422-8c5b7a147e24">